test/module/irohad/model/command_validate_execute_test.cpp

```diff
@@ -660,7 +656,8 @@ TEST_F(CreateAssetTest, InvalidWhenNoPermissions) {
  * @then execute() fails
  */
 TEST_F(CreateAssetTest, InvalidWhenAssetInsertionFails) {
-  EXPECT_CALL(*wsv_command, insertAsset(_)).WillOnce(Return(false));
+  EXPECT_CALL(*wsv_command, insertAsset(_))
+      .WillOnce(Return(WsvCommandResult(iroha::expected::makeError(""))));
```

Comment:
Would be nice to have a shortcut on that

Reply:
It would be great to check error messages as well, but that's beyond the scope of this pr.

And yes, for now we need some shortcut